dspace-spring-rest/src/main/resources/application.properties

```diff
@@ -77,7 +77,7 @@ server.port=8080
 
 # This creates a Tomcat context-param named "dspace.dir"
 # and sets it to the value of the "dspace.dir" property (listed above)
-server.context-parameters.dspace.dir=${dspace.dir}
+server.context-parameters.dspace.dir=/opt/dspace
```

Comment:
please remote this local configuration

Reply:
Sorry for the local configurations. I'll remove them from the PR.